Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js

```diff
@@ -110,7 +138,10 @@ var deleteIncidents = function(ids_to_delete) {
         throw res[0].Contents;
     }
 
-    var response = res['response']
+    var response = res['response'];
+    if (fields_to_keep != "all") {
```

Comment:
This will help handling tasks that were set with previous argument value.
```suggestion
    if (fields_to_keep && (fields_to_keep != "all")) {
```

Reply:
This is correct when running from the `CLI`, or when setting up a new `playbook`, however there's one edge case this doesn't cover - `playbook` task that was configured before the change.
This is why it's best to add this change.